ADVANCED.md

```diff
@@ -949,14 +949,14 @@ Publish new version
 
 1. Create new version branch `git checkout -b v0.6`.
```

Comment:
Does this line also have to be `v0.6.0` to follow the rest of the version updates?

Reply:
That is actually intentionally "v0.6". That branch will track the latest version of v0.6, which will be a branch that is maintained in parallel to master.